web-client/src/presenter/actions/PendingItems/setTitleForGlobalReportAction.js

```diff
@@ -3,10 +3,17 @@ import { state } from 'cerebral';
 /**
  * Sets the title for the screen
  *
- * @param {object} providers the providers object
- * @param {object} providers.store the cerebral store object used for setting pendingItems
- * @param {object} providers.props the pendingItems to set
+ * @param {string} headerTitle the name of the header title
+ * @returns {Promise} async action
  */
-export const setTitleForGlobalReportAction = ({ store }) => {
-  store.set(state.screenMetadata.headerTitle, 'Pending Report');
-};
+export const setTitleForGlobalReportAction =
+  headerTitle =>
+  /**
+   * sets the state.screenMetaDate.headerTitle based on the scoped page
```

Comment:
Nit: screenMetaData sp

Reply:
It also looks like the jsdocs could be updated :)

Reply:
Done